besu/src/main/java/org/hyperledger/besu/cli/options/DataStorageOptions.java

```diff
@@ -52,45 +52,46 @@ public class DataStorageOptions implements CLIOptions<DataStorageConfiguration>
   private DataStorageFormat dataStorageFormat = DataStorageFormat.BONSAI;
 
   @Option(
-      names = {BONSAI_STORAGE_FORMAT_MAX_LAYERS_TO_LOAD, "--bonsai-maximum-back-layers-to-load"},
+      names = {DIFFBASED_STORAGE_FORMAT_MAX_LAYERS_TO_LOAD, "--bonsai-maximum-back-layers-to-load"},
```

Comment:
feels odd that the option is generic but then the cli option is specific to bonsai. Are you going to use the cli option to configure verkle too?

Reply:
In the future, I will add a second option for Verkle, --verkle-maximum-back-layers-to-load, but in the end, it will be the same parameter in the code. I think for the user experience, it's better to have --verkle-maximum-back-layers-to-load and --bonsai-maximum-back-layers-to-load instead of --diffbased-maximum-back-layers-to-load because users don't know what that is.